Modules/Filtering/DistanceMap/test/itkHausdorffDistanceImageFilterAnisotropicCorrectness.cxx

```diff
@@ -0,0 +1,107 @@
+/*=========================================================================
+ *
+ *  Definitive test showing ITK correctly handles anisotropic spacing
+ *
+ *=========================================================================*/
+
+#include "itkHausdorffDistanceImageFilter.h"
+#include "itkTestingMacros.h"
+
+int
+itkHausdorffDistanceImageFilterAnisotropicCorrectness(int, char *[])
+{
+  constexpr unsigned int ImageDimension{ 3 };
+
+  using PixelType = unsigned char;
+  using ImageType = itk::Image<PixelType, ImageDimension>;
+
+  std::cout << "\n=== Definitive Test: ITK Correctly Handles Anisotropic Spacing ===" << std::endl;
+  std::cout << "\nScenario: Two points separated in z-direction only" << std::endl;
+
+  int exitStatus = EXIT_SUCCESS;
+
+  // Create simple test: two single voxels
+  auto image1 = ImageType::New();
+  auto image2 = ImageType::New();
+
```

Comment:
Please consider using GoogleTest for these kinds of tests.

When a unit test just creates its input data on the fly (in memory), and when it does not need any external parameters (as its function parameters `(int, char *[])` are unused), it is often preferred to use GoogleTest.

GoogleTest unit tests are typically smaller, as each of them just tests a specific feature or a specific property.

Reply:
Unless Matt has a different opinion, we can close this PR. This class already had 2 tests.